### ISSUE TITLE
Move the response encryption key remark outside of the new parameters list

### ISSUE THREAD

**Issue by yaromin** [openid/OpenID4VP#176]
This fragment cuts the list into two fragments. It is unclear whether the `request_uri_method` that follows is still part of the same list and why the description of something that is not a new parameter appears here.

> A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter. 

I would reword it slightly ...

_If encryption of the Authorization Response is needed, the public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)) MAY be passed by the Verifier using the jwks or the jwks_uri claim in the Client Metadata._

... and move it after the `request_uri_method`. 
https://github.com/openid/OpenID4VP/blob/9a8ac76f93d0240767d92152085ae08c420d71cc/openid-4-verifiable-presentations-1_0.md?plain=1#L276

Any opinions?

**Comment by jogu** [comment]
Resolved by https://github.com/openid/OpenID4VP/pull/233 - this is how it now looks - the indentation is fixed and the language around the jwks contents has been significantly reworked:

<img width="783" alt="Screenshot 2024-09-27 at 09 42 25" src="https://github.com/user-attachments/assets/6b9b9776-bf42-4d45-a166-ea0c95a65bc2">
